doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -175,19 +177,82 @@ The NFVI certification framework will be guided by the following core principles
 -   Add test cases from operators, which operators already tested in their environment
 
 <a name="2.7.2"></a>
-### 2.7.2 Testing
+### 2.7.2 Test case integration requirements
+
+[Xtesting](https://xtesting.readthedocs.io/en/latest/) is a simple framework to
+assemble sparse test cases and to accelerate the adoption of CI/CD best
+practices. By managing all the interactions with the CI/CD components (test
+scheduler, test results database, artifact repository), it allows the developer
+to work only on the test suites without diving into CI/CD integration. Even
+more, it brings the capability to run heterogeneous test cases in the same CI
+toolchains thanks to a few low constraints
+[quickly achievable](https://www.sdxcentral.com/articles/news/opnfvs-6th-release-brings-testing-capabilities-that-orange-is-already-using/2018/05/).
+
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) is a more recent
```

Comment:
@collivier this is already covered in the cookbook where it belongs: https://github.com/cntt-n/CNTT/pull/873 this chapter is intended to be for requirements without specifying specific projects. i suggest removing references to projects

Reply:
@rabiabdel For my understanding we should ask as prerequisite the common testcase execution as proposed by Xtesting. I can move the XtestingCI part as described here in Cookbook (see https://github.com/cntt-n/CNTT/pull/873) but I would consider that Xtesting should stay here.
We won't be able to integrate all the testcases in a smooth way without a testcase execution.
Is it ok? then I will update the 2 PRs accordingly.

Reply:
@collivier if this chapter focuses on the requirement to have test case execution environment  and the cookbook specify XtextingCI and Xtesting that would be ideal. this way it will flow better. i hope this is OK.

Reply:
Xtesting is by design already in-use by Functest  and has been highlighted by OPNFV Marketing 2020 for the common OPNFV testcase execution. Let's see if I can rephrase it and stay precise too.

Reply:
@rabiabdel here is a new proposal. Hoping it has been improved enough and takes into account most of your comments (at least XtestingCO).
I think I can't simply remove Xtesting because the cookbook already in use leverages on it as well as the smooth integration of the next testcases in the current RI and RC toolchains (then I would think it has to be introduced before 9.2 and 2.7 Frameworks seems the best place).
From my understanding, the document will be modified if we decide to switch to another framework meeting the same requirements mentioned in the first part in a better way (RI and RC cookbook would be modified too).